timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -735,6 +735,7 @@ def fit_ensemble(
         )
 
         self._add_model(model=simple_ensemble, base_models=simple_ensemble.model_names)
+        self.save_model(model=simple_ensemble)
```

Comment:
What was happening before this change?

Reply:
The `WeightedEnsemble` model is saved [inside the model graph](https://github.com/autogluon/autogluon/blob/master/timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py#L175) (i.e., as an attribute of the trainer saved to `trainer.pkl`). This means that we could predict with the ensemble after loading the predictor from disk. 

The new line `self.save_model(model=simple_ensemble)` saves the metadata of the model (val_score, fit_time, etc.) to disk so that it can be retrieved when calling `info()` and `get_summary()`.